src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -128,11 +128,11 @@ function executeCommand(command) {
         'image/x-bil;bits=32': getXbilTexture,
     };
 
-    const func = supportedFormats[layer.options.mimetype];
+    const func = supportedFormats[layer.format];
     if (func) {
         return func(tile, layer, command.targetLevel);
     } else {
-        return Promise.reject(new Error(`Unsupported mimetype ${layer.options.mimetype}`));
+        return Promise.reject(new Error(`Unsupported mimetype ${layer.format}`));
```

Comment:
Last thing I can see, maybe change the mimetype word here too (present in WFS/Static/WMTS/WMS). Otherwise it is good to me.